.github/workflows/build-ampd-main.yaml

```
@@ -1,17 +1,14 @@
-name: Build and push latest docker image
+name: Amplifier - Build main branch
```

```suggestion
name: Amplifier - main branch auto-builder
```
perhaps to better distinguish auto build actions vs manually triggered actions?

---

its both auto build and can be manually triggered.

---

Fair but the main purpose is auto build anyways

---

yes but then ill get questions about how to trigger the auto builder manually.

---

I'm with Talal on this. If I see auto-builder I will ignore it in the search for the correct workflow